data/hai/hai reveal 1 intro.txt

```diff
@@ -36,7 +36,6 @@ mission "Hai Reveal: Reputation Reset"
 		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
 		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
 		# Make sure the new governments start with appropriate rep:
-		"reputation: Hai (Friendly Unfettered)" = 1000
 		"reputation: Elenctic Commune" = 1000
 		"reputation: Pirate (Devil-Run Gang)" = -1000
```

Comment:
For the record: this is where the player reputation with this government starts, and all of its attitudes are low enough that it won't change, so this line is also not at all necessary.

Reply:
I was under the impression this was not the case for existing saves - that is the reasoning given to me for why it was put there.

Reply:
Easy then
```suggestion
```